pettingzoo/utils/wrappers/supersuit/basic_wrappers.py

```diff
@@ -0,0 +1,95 @@
+from reward_lambda import reward_lambda_v0, AecRewardLambda
+from observation_lambda import observation_lambda_v0, AecObservationLambda
+from utils.basic_transforms import color_reduction
+from typing import Literal, Any
+from types import ModuleType
+from pettingzoo import AECEnv, ParallelEnv
+from gymnasium.spaces import Space
+import numpy as np
+
+
+def basic_obs_wrapper(env: AECEnv | ParallelEnv, module: ModuleType, param: Any) -> AecObservationLambda:
+    """
+    Wrap an environment to modify its observation space and observations using a specified module and parameter.
+
+    This function takes an environment, a module, and a parameter, and creates a new environment with an observation
+    space and observations modified based on the provided module and parameter.
+
+    Parameters:
```

Comment:
If you could have the docstrings be more similar to the syntax in `conversions.py` and other wrappers in PettingZoo that would be good. Also I don't know the wrapper super well but I think `my_module` and `my_param` may not be the correct names. In general using GPT to write this stuff is pretty risky because it could very well just be completely made up and I don't have the time to look through all of the specifics to ensure it's correct. If you can do so yourself and double check then that's great but I'm hesitant to include too much details because it could be incorrect. Look elsewhere throughout the repo to see how the formatting is done for the docstrings.

And for the example format, see chess.py as we have an example using the `>>>` format which gets tested under the doctests with pytest.

Reply:
Ok thanks for the advice, I'll fix the docstring format and write them from scratch.

Reply:
Agree, definitely not use the output of CGPT directly for docstrings.